simtools/model/camera.py

```diff
@@ -0,0 +1,670 @@
+                        'defaulting to 4.9 cm')
+            pixels['diameter'] = 4.9
+
+        return pixels
+
+    def _rotatePixels(self, telescopeType, pixels):
+        '''
+        Rotate the pixels according to the rotation angle given in pixels['rotateAngle'].
+        Additional rotation is added to get to the camera view of an observer facing the camera.
+        The angle for the axes rotation depends on the coordinate system in which the original
+        data was provided.
+
+        Parameters
+        ----------
+        telescopeType: string
+            As provided by the telescope model method "TelescopeModel".
+        pixels: dictionary
+            The dictionary produced by the readPixelList method of this class
+
+        Returns
+        -------
+        pixels: dict
+            The pixels dictionary with rotated pixels.
+            The pixels orientation for plotting is added to the dictionary in pixels['orientation'].
+            The orientation is determined by the funnel shape (see readPixelList for details).
+
+        Notes
+        -----
+        The additional rotation angle to get to the camera view of an observer facing the camera
+        is saved in the const dictionary CAMERA_ROTATE_ANGLE.
+        In the case of dual mirror telescopes, the axis is flipped in order to keep the same
+        axis definition as for single mirror telescopes.
+        The list of dual mirror telescopes is given in the const dictionary TWO_MIRROR_TELS.
+        '''
+
+        if telescopeType not in TWO_MIRROR_TELS:
+            pixels['y'] = [(-1)*yVal for yVal in pixels['y']]
+
+        rotateAngle = pixels['rotateAngle']  # So not to change the original angle
+        rotateAngle += np.deg2rad(CAMERA_ROTATE_ANGLE[telescopeType])
+
+        logger.debug('Rotating pixels by {}'.format(np.rad2deg(rotateAngle)))
+
+        if rotateAngle != 0:
+            for i_pix, xyPixPos in enumerate(zip(pixels['x'], pixels['y'])):
+                pixels['x'][i_pix] = xyPixPos[0]*np.cos(rotateAngle) - xyPixPos[1]*np.sin(rotateAngle)
+                pixels['y'][i_pix] = xyPixPos[0]*np.sin(rotateAngle) + xyPixPos[1]*np.cos(rotateAngle)
+
+        pixels['orientation'] = 0
+        if pixels['funnelShape'] == 1 or pixels['funnelShape'] == 3:
+            if pixels['funnelShape'] == 3:
+                pixels['orientation'] = 30
+            if rotateAngle > 0:
+                pixels['orientation'] += np.rad2deg(rotateAngle)
+
+        return pixels
+
+    def calcFOV(self):
+        '''
+        Calculate the FOV of the camera in degrees, taking into account the focal length.
+
+        Returns
+        -------
+        fov: float
+            The FOV of the camera in the degrees.
+        averageEdgeDistance: float
+            The average edge distance of the camera
+
+        Notes
+        -----
+        The x,y pixel positions and focal length are assumed to have the same unit (usually cm)
+
```

Comment:
I don't use these blank lines in the end of docstrings. We should decide for using it or not.

Reply:
Wasn't intentional, removed.